### ISSUE TITLE
How to evaluate performance on test set of MOT16/MOT17 without ground truths ? 

### ISSUE THREAD

**Issue by farooqrasheed123** [ifzhang/FairMOT#268]
How can we evaluate MOT16/MOT17 test data as we don't have ground truth for that? During training, we have ground truths and we can evaluate results on training data but how to evaluate on test data of MOT16 or MOT17 without ground truths ?

**Comment by Ravsehajsinghpuri** [comment]
Hi, there is no way to evaluate the results without ground truth. What you need to do is simply submitting the files to the evaluation server of MOTchallenge after registering and logging in. Here is the link to the challenge. https://motchallenge.net

**Comment by farooqrasheed123** [comment]
Thanks
